python/ray/_raylet.pyx

```diff
@@ -264,23 +264,31 @@ class ObjectRefGenerator:
 
 
 class StreamingObjectRefGenerator:
```

Comment:
Annotate with @PublicAPI

Also do we want to start with alpha or beta or stable

Reply:
This is the scope of the second PR!

Reply:
We will scope it to beta